protonvpn_cli/utils.py

```diff
@@ -19,8 +19,12 @@
 )
 
 
-def call_api(url, json_format=True):
-    """Call to the ProtonMail API at https://api.protonmail.ch."""
+def call_api(endpoint, json_format=True, handle_errors=True):
+    """Call to the ProtonVPN API."""
+
+    api_domain = "https://api.protonmail.ch"
```

Comment:
It must be api.protonvpn.ch

Reply:
It really would be better to make it configurable.

Reply:
As discussed in #42 :)